build.yaml

```diff
@@ -21,6 +21,8 @@ release:
       each:
         - script: |
             LIBUV_VERSION=1.14.1
+            CC=g++
```

Comment:
This can be `gcc` instead of `g++`.

Reply:
Should `CC` be an environment variable instead for the matrix (might be overkill though)?

Reply:
All our snapshots currently use preinstalled gcc. We can change it to a matrix env if we enable CI and want to build against different compilers.